configs/bsc/wsteth-mainnet.yml

```diff
@@ -0,0 +1,666 @@
+    - &l2AxelarTransceiver "0x723AEAD29acee7E9281C32D11eA4ed0070c41B13"
+    - &l2AxelarTransceiverImpl "0xa1ebb6a4b856df8bf6c3aca88a9115a9ab3b2e02"
+    # Stateless libraries
+    # &l2TransceiverStructs "0xf0396a8077eda579f657B5E6F3c3F5e8EE81972b"
+    # &l2TransceiverStructsAxelar "0x27a3daf3b243104e9b0afae6b56026a416b852c9"
+
+l1:
+  rpcUrl: https://ethereum-rpc.publicnode.com
+  explorerHostname: api.etherscan.io
+  explorerTokenEnv: ETHERSCAN_TOKEN
+  contracts:
+    l1NttManager:
+      name: NttManager
+      address: *l1NttManager
+      proxyName: ERC1967Proxy
+      implementation: *l1NttManagerImpl
+      proxyChecks: {}
+      checks:
+        NTT_MANAGER_VERSION: *NTT_MANAGER_VERSION # 1.1.0
+        chainId: *ETH_CHAIN_ID
+        getCurrentInboundCapacity:
+          - args: [ *ETH_CHAIN_ID ]
+            result: 0
+          - args: [ *BSC_CHAIN_ID ]
+            result: *RATE_LIMIT
+        getCurrentOutboundCapacity: *RATE_LIMIT # 30000000000000000000000
+        getInboundLimitParams:
+          - args: [ *ETH_CHAIN_ID ]
+            result: [0, 0, 0]
+          - args: [ *BSC_CHAIN_ID ]
+            result: [768000000000008, 768000000000008, 1722437087]
+        getInboundQueuedTransfer:
+          - args: [ *ZERO_BYTES32 ]
+            result: [ 0, 0, *ZERO_ADDRESS ]
+        getMigratesImmutables: false
+        getMode: *LOCKING_MODE
+        getOutboundLimitParams: [768000000000008, 767999999974408, 1722437087]
+        getOutboundQueuedTransfer:
+          - args: [ 0 ]
+            result: [ *ZERO_BYTES32, *ZERO_BYTES32, 0, 0, 0, *ZERO_ADDRESS, "0x" ]
+        getPeer:
+          - args: [ *ETH_CHAIN_ID ]
+            result: [ *ZERO_BYTES32, 0 ]
+          - args: [ *BSC_CHAIN_ID ]
+            result: [ *L2_NTT_MANAGER_BYTES32_ADDR, *DECIMALS ]
+        getThreshold: *THRESHOLD # 2
+        getTransceiverInfo: [[ true, true, 0 ],[ true, true, 1 ]] # registered, enabled, index
+        getTransceivers: [ *l1WormholeTransceiver, *l1AxelarTransceiver ] # 0xA1ACC1e6edaB281Febd91E3515093F1DE81F25c0,0x723AEAD29acee7E9281C32D11eA4ed0070c41B13
+        isMessageApproved:
+          - args: [ *ZERO_BYTES32 ]
+            result: false
+        isMessageExecuted:
+          - args: [ *ZERO_BYTES32 ]
+            result: false
+        isPaused: false
+        messageAttestations:
+         - args: [ *ZERO_BYTES32 ]
+           result: 0
+        mode: *LOCKING_MODE
+        nextMessageSequence: 38
+        owner: *L1_NEW_INTERIM_MSIG
+        pauser: *L1_NEW_INTERIM_MSIG
+        quoteDeliveryPrice:
+          - args: [ *BSC_CHAIN_ID, 0x ]
+            result: [[], 0]
+            mustRevert: true
+        rateLimitDuration: *RATE_LIMIT_DURATION
+        token: *L1_WSTETH
+        tokenDecimals: *DECIMALS # 18
+        transceiverAttestedToMessage:
+          - args: [ *ZERO_BYTES32, 0 ]
+            result: false
+          - args: [ *ZERO_BYTES32, 1 ]
+            result: false
+      implementationChecks:
```

Comment:
Do we need to check all the same methods? You can validate only those that should differ (e.g. adi implementation checks). This will drastically reduce the number of checks and make verification simpler by focusing on those methods that will vary from the proxied version.

Reply:
Example: 
https://github.com/lidofinance/state-mate/blob/main/configs/binance/mainnet.yml#L158-L161
implementation checks only validate those methods that are crucial to be different from the proxied version.

Reply:
I will use it for future verification efforts for sure, thanks! :+1:
For now I'd prefer to leave at as is